cmd/dep/remove_test.go

```diff
@@ -110,53 +84,33 @@ func main() {
 		t.Fatal("rm of arg in manifest and imports should have failed without -force")
 	}
 
-	h.TempFile("src/"+root+"/manifest.json", origm)
+	h.TempCopy(root+"/manifest.json", "remove/manifest.input.json")
 	h.Run("remove", "-force", "github.com/pkg/errors", "github.com/not/used")
 
-	manifest = h.ReadManifest()
-	if manifest != `{
-    "dependencies": {
-        "github.com/Sirupsen/logrus": {
-            "revision": "42b84f9ec624953ecbf81a94feccb3f5935c5edf"
-        }
-    }
-}
-` {
-		t.Fatalf("expected %s, got %s", expectedManifest, manifest)
+	goldenManifest = "remove/manifest1.golden.json"
+	wantManifest = h.GetTestFileString(goldenManifest)
+	gotManifest = h.ReadManifest()
+	if wantManifest != gotManifest {
+		if *test.UpdateGolden {
+			if err := h.WriteTestFile(goldenManifest, gotManifest); err != nil {
+				t.Fatal(err)
+			}
+		} else {
+			t.Errorf("expected %s, got %s", wantManifest, gotManifest)
+		}
 	}
 
 	sysCommit := h.GetCommit("go.googlesource.com/sys")
-	expectedLock := `{
-    "memo": "7769242a737ed497aa39831eecfdc4a1bf59517df898907accc6bdc0f789a69b",
-    "projects": [
-        {
-            "name": "github.com/Sirupsen/logrus",
-            "revision": "42b84f9ec624953ecbf81a94feccb3f5935c5edf",
-            "packages": [
-                "."
-            ]
-        },
-        {
-            "name": "github.com/pkg/errors",
-            "version": "v0.8.0",
-            "revision": "645ef00459ed84a119197bfb8d8205042c6df63d",
-            "packages": [
-                "."
-            ]
-        },
-        {
-            "name": "golang.org/x/sys",
-            "branch": "master",
-            "revision": "` + sysCommit + `",
-            "packages": [
-                "unix"
-            ]
-        }
-    ]
-}
-`
-	lock := h.ReadLock()
-	if lock != expectedLock {
-		t.Fatalf("expected %s, got %s", expectedLock, lock)
+	goldenLock := "remove/lock1.golden.json"
+	wantLock := strings.Replace(h.GetTestFileString(goldenLock), "` + sysCommit + `", sysCommit, 1)
```

Comment:
It's not great that we're still having to do this. I've opened #255 to get rid of it.